yw_website/website/templates/account/base.html

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
what is this template for? are we straying from the `extendable`/`includeable`/`pages` file structure format?

Should we be inheriting from `__base.html` ?